org.eclipse.jdt.ui.tests.refactoring/test cases/org/eclipse/jdt/ui/tests/refactoring/ExtractTempTests.java

```diff
@@ -876,6 +876,43 @@ public void test125() throws Exception {
 		helper1(9, 32, 9, 42, true, false, "length", "length");
 	}
 
+
```

Comment:
You should add a test that has a method with parameter(s) and have the parameter expression be a prefix expression so it cannot be done in multiple.  You should also add another test which is the same code but doesn't call with a prefix expression so that all calls are replaced.